src/UniGet/RemoveTool.cs

```diff
@@ -106,8 +106,8 @@ internal static bool RemoveEmptyFolder(string path)
 
             if (ok && Directory.GetFiles(path).Any() == false)
             {
+                var metaFile = Path.GetFullPath(Path.Combine(path, "..", Path.GetFileName(path))) + ".meta";
```

Comment:
Sorry, for the delay answering. I think that I pressed the wrong button and the reply was stuck in a pending mode. :-(